app/src/main/java/de/westnordost/streetcomplete/quests/wheelchair_access/AddWheelchairAccess.java

```diff
@@ -0,0 +1,66 @@
+package de.westnordost.streetcomplete.quests.wheelchair_access;
+
+import android.os.Bundle;
+
+import javax.inject.Inject;
+
+import de.westnordost.streetcomplete.data.QuestImportance;
+import de.westnordost.streetcomplete.data.osm.SimpleOverpassQuestType;
+import de.westnordost.streetcomplete.data.osm.changes.StringMapChangesBuilder;
+import de.westnordost.streetcomplete.data.osm.download.OverpassMapDataDao;
+import de.westnordost.streetcomplete.quests.AbstractQuestAnswerFragment;
+import de.westnordost.streetcomplete.quests.YesNoQuestAnswerFragment;
+import de.westnordost.streetcomplete.quests.opening_hours.AddOpeningHoursForm;
+
+
+public class AddWheelchairAccess extends SimpleOverpassQuestType
+{
+	@Inject public AddWheelchairAccess(OverpassMapDataDao overpassServer)
+	{
+		super(overpassServer);
+	}
+
+	@Override protected String getTagFilters()
+	{
+		return " nodes, ways, relations with ( shop and shop !~ no|vacant or" +
+				" amenity ~ restaurant|cafe|ice_cream|fast_food|bar|pub|biergarten|food_court|cinema|nightclub|" +
+				"bank|atm|bureau_de_change|money_transfer|post_office|library|courthouse|embassy|" +
+				"car_wash|car_rental|marketplace|fuel|driving_school|" +
+				"dentist|doctors|clinic|pharmacy|veterinary or" +
+				" amenity = bicycle_parking and bicycle_parking = building or" +
+				" amenity = parking and parking = multi-storey or" +
+				" amenity = recycling and recycling_type = centre or" +
+				" tourism ~ zoo|aquarium|theme_park|gallery|museum or" +
+				" tourism = information and information = office or" +
+				" tourism ~ hotel|guest_house|hostel|motel or" +
+				" leisure ~ golf_course|water_park|miniature_golf|dance|bowling_alley|horse_riding" +
+				"sports_centre|fitness_centre|amusement_arcade|adult_gaming_centre|tanning_salon )" +
```

Comment:
To circumvent mistakes like this, one could also do it like this:
```
String[] LEISURES= {"golf_course","water_park",.... etc};
and then use TextUtils.join("|",LEISURES);
```
Just a suggestion, up to you which you find better. I did this with some, but not all. **IF** you change it, could you also do the same for opening hours so these two are consistent?